hydradx/model/amm/stableswap_amm.py

```diff
@@ -108,16 +112,45 @@ def calculate_y(self, reserves: list, d: float, max_iterations=128):
         return y
 
     # price is denominated in the first asset
-    @property
-    def spot_price(self):
-        x, y = self.liquidity.values()
-        return self.price_at_balance([x, y], self.d)
+    def spot_price(self, i: int):
```

Comment:
for backward compatibility can we set default i = 1?

Reply:
done